src/utils/messages.ts

```diff
@@ -0,0 +1,6 @@
+const MESSAGES = {
+  DEPRECATION_MESSAGE:
```

Comment:
I would call it only `DEPRECATION`  (since this is nested under `Messages` already) or `DEPRECATED_FUNCTION`.
`Messages.DEPRECATED_FUNCTION`